journey/app/models/profile.rb

```diff
@@ -1,3 +1,3 @@
 class Profile < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, inverse_of: :profile, required: true
```

Comment:
- By default belongs_to validates the presence of the associated record to guarantee reference consistency. Hence we can omit `required: true`.

- Also Rails tries to detect inverse for `has_many`, `has_one` and `belongs_to` that do not have `through`, `foreign_key` options and scope and have standard names and `inverse_of: :profile` can also get omitted.  
```suggestion
  belongs_to :user
```